### DISCUSSION TITLE
Update Neovide for Windows

### DISCUSSION THREAD

**Discussion by anson-ryea** [neovide/neovide#1040] [Q&A] [answered]
Is there any way I can update Neovide to 1.0 in scoop? the latest version there is 0.5. I have been trying to build from source but nothing works.

**Comment by dineshKumar777** [comment, marked as answer]
you can simply download the executable from actions page.
https://github.com/neovide/neovide/actions/runs/1396712234

**Reply by anson-ryea** [reply]
Argh, I should have made a thread earlier. Spent a few hours struggling with that.
